scripts/README.md

```diff
@@ -14,46 +14,83 @@ npx truffle compile
 yarn run networks-inject
 ```
 
-Setup the env variables:
+Create a gnosis-safe wallet [here](https://rinkeby.gnosis-safe.io). This wallet will be called your Master Safe in the following. It is used to bundle the transactions and setup the bracket-traders.
+This Master Safe must have an additional owner `0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1`, known as the "Proposer" account. The following scripts will use this account to propose transactions to the interface. This implies that the mnemonic phrase for this "Proposer" account is stored in plain text within this project.
+
+In order to have a save setup, make sure that _your Master Safe always requires one more signature than just the signature of the Proposer account to send a transaction_. _Otherwise, everyone can steal the funds from your account!_
+
+Setup env variables for the deployment process:
 
 ```
 export PK=<Your Key>
-export GAS_PRICE_GWEI=8
+export GAS_PRICE_GWEI=<look up the suggestion from ethgasstation.info>
 export NETWORK_NAME=<network>
 export MASTER_SAFE=<master safe>
 
 ```
 
-### Confirming multisig orders with Metamask
+### Confirming multisig-transactions on gnosis-safe with Metamask
+
+For the signing process, note that the gas consumption is underestimated. There is currently no proper way to estimate the gas limit. In order to make sure that the gas limit of your transaction is sufficient, increase it to 5 million gas. In practice, this limit suffices to deploy a strategy with 20 brackets.
```

Comment:
We could make an educated guess, however. Do we want to have a formula here? The only issue is that some tokens are more expensive than others, I think we used more than 5M sometimes.

Reply:
I think one of our most expensive ones was this one:
https://etherscan.io/tx/0xe1853d4a57b6b46ca89df4f9725b8230234c2f173f940fc5372a15261291d164
or this one with 22 brackets:
https://etherscan.io/tx/0xf1a55141c30d4d604fcb712a0ab5e5d2b823ca75ff21777d7a88960840b8ebb6#eventlog


But I can change it to 5.9 million. I just remember that there was some issue with "rinkeby?" or another network when we send more than 6 million gas.

Reply:
True, we have to be careful in Rinkeby, not sure what the exact limit is however. 5 looks tight to me, I'm ok with 5.9 or even 5.5.